docs/hadoop-catalog.md

```diff
@@ -104,19 +105,27 @@ The Hadoop catalog supports creating, updating, deleting, and listing schema.
 
 ### Schema properties
 
-| Property name                         | Description                                                                                                    | Default value             | Required | Since Version    |
-|---------------------------------------|----------------------------------------------------------------------------------------------------------------|---------------------------|----------|------------------|
-| `location`                            | The storage location managed by Hadoop schema.                                                                 | (none)                    | No       | 0.5.0            |
-| `authentication.impersonation-enable` | Whether to enable impersonation for this schema of the Hadoop catalog.                                         | The parent(catalog) value | No       | 0.6.0-incubating |
-| `authentication.type`                 | The type of authentication for this schema of Hadoop catalog , currently we only support `kerberos`, `simple`. | The parent(catalog) value | No       | 0.6.0-incubating |
-| `authentication.kerberos.principal`   | The principal of the Kerberos authentication for this schema.                                                  | The parent(catalog) value | No       | 0.6.0-incubating |
-| `authentication.kerberos.keytab-uri`  | The URI of The keytab for the Kerberos authentication for this schema.                                         | The parent(catalog) value | No       | 0.6.0-incubating |
-| `credential-providers`                | The credential provider types, separated by comma.                                                             | (none)                    | No       | 0.8.0-incubating |
+| Property name                         | Description                                                                                                                 | Default value             | Required | Since Version    |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|---------------------------|----------|------------------|
+| `location`                            | The storage location managed by Hadoop schema. It's location name is `unknown`.                                             | (none)                    | No       | 0.5.0            |
+| `location-`                           | The property prefix. User can use `location-{name}=location` to set multiple locations with different names for the schema. | (none)                    | No       | 0.9.0-incubating |
```

Comment:
There will be a misunderstanding that the `location` here in `location-{name}=location` has something related to the above `location` property, you should use a different name to differentiate.

Reply:
fixed